File: 1ºAno/Lista2/C/Sala/files/ex9.c
#include <stdio.h>

int main() {
    
    int a, b;
    printf ("Entre com 2 números, dir-te-ei se há divisibilidade por 4 ou 5:\n");
    printf ("N1 = ");
    scanf ("%d", &a);
    printf ("N2 = ");
    scanf ("%d", &b);
    if (a % 4 == 0 || a % 5 == 0) {
        printf ("%d é divisível por 4 ou 5\n", a);
    }
    if (b % 4 == 0 || b % 5 == 0) {
        printf ("%d é divisível por 4 ou 5\n", b);
    }
    
    return 0;
}

File: 1ºAno/Lista2/Python/Sala/files/ex9.py
nums = input("Entre com os números: ").split()
for i in nums:
    if int(i) % 4 == 0 or int(i) % 5 == 0:
        print (i,"",end='')
print ("atende(m) os critérios de divisibilidade ou por 4 ou por 5")